website/docs/r/lb_listener_rule.html.markdown

```diff
@@ -97,6 +97,67 @@ resource "aws_lb_listener_rule" "health_check" {
 
 ```
 
+### Authenticate-cognito Action
```

Comment:
Same comments for these examples as well.

Reply:
Yes, I just noticed that the documentation in `lb_listener_rule.html.markdown` is in a bit different format. I was updating this while you've been reviewing. 😄 Should be 👌 now, can you check again?